src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionEdit.tag

```diff
@@ -23,7 +23,22 @@
                     <h3 class="text-center">Previewing Session as ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})</h3>
                 </c:when>
                 <c:when test="${data.moderation}">
-                    <h3 class="text-center">Moderating Responses for ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})</h3>
+                    <div class="container">
+                        <div class="col-md-12">
+                            <h3 class="text-center">
+                              You are moderating responses for (or submitting on behalf of) ${isInstructor ? "Instructor" : "Student"} ${moderatedPersonName} (${moderatedPersonEmail})
```

Comment:
@JosephineKwa 
Sure, I'll change. But is it necessary to update the HTML coding standard [documentation](https://oss-generic.github.io/process/codingStandards/CodingStandard-Html.html) in the Miscellaneous section. It actually uses two spaces for content.